### ISSUE TITLE
How to setup UPBGE?

### ISSUE THREAD

**Issue by cederom** [UPBGE/blender#1082]
It is unclear on how to setup UPBGE. No mention in the README nor docs :-(

Is it a total fork of Blender?

Is it a Blender Add-On that needs to be compiled and then installed in new Blender 2.80+ official release?

**Comment by marechal-p** [comment]
It is a whole fork of Blender. For instructions about building it, see the wiki: https://github.com/UPBGE/blender/wiki/User-notes

**Comment by cederom** [comment]
Thank you @marechal-p :-) Would that be possible to rename it so it does not collide with current blender installation on a POSIX OS like FreeBSD i.e. I could have both UPBGE and Blender on the same machine? :-)

**Comment by marechal-p** [comment]
I don't remember well, but either the build already outputs something like `upbge` and `upbgeplayer` binaries, and if not you can always rename them after :)

**Comment by youle31** [comment]
This needs to be build using git, cmake and compilation tutorials:

I answered here: https://github.com/UPBGE/blender/issues/1081

Yes, this is a complete fork of blender. As far I know, there is no issue to have blender official and upbge side by side

**Comment by cederom** [comment]
Thank you guys for hints and your great project!!!

BGE removal from Blender, which was its core assumption and functionality, is a disaster decission!

I was using Blender for REAL TIME VR AND ENGINEERING not animation or graphics.

I hope they will review their decission and bring back GAME MODE in any way. I know BGE was pain in the ass but it allowed really cool works to be done in this great Blender environment having everything at hand and absolutely no external deendencies. Now Blender is __just__ a graphics and animation tool.

**Comment by cederom** [comment]
I still have some questions, these may come handy for other porters as well :-)
* I will create a `make.sh` script to preform all steps reported in various threads to automate build on FreeBSD and Linux.
* Is tag `v0.2.4`  latest stable release of UPBGE? I would prefer to work on stable stuff first :-)
* Is branch `upbge0.3` current development head/master?
* Why project is still named `blender` not `upbge`?
* Why there are still branches and tags from original `blender`? No `blender` nor `upbge` prefix. This brings confusion.

When things work out I will create a dedicated `Port` that is `Makefile` that fully automates download/build/install procedure on FreeBSD. This has some restrictions and limitations, which makes things well organized and secure. This may work out also for Linux (i.e. build script cannot modify system with sudo, instead it should work in a temporary work directories and generate results in one common place, etc).

**Comment by marechal-p** [comment]
> Why project is still named `blender` not `upbge`?

I think they just forked and didn't rename at the time, then sticked to it?

> Why there are still branches and tags from original `blender`? No `blender` nor `upbge` prefix. This brings confusion.

GitHub forks do this, no-one really tried to clean, maybe @youle31 even brought some over to keep working on EEVEE, not sure.

> Is branch `upbge0.3` current development head/master?

From the looks of it, I would say yes. @youle31 and @lordloki are having fun porting the BGE to Blender 2.8 :)

> Is tag `v0.2.4` latest stable release of UPBGE? I would prefer to work on stable stuff first :-)

It was the latest tag from when @panzergame was still working on UPBGE, but this specific release is a bit bogus, even though I like that it ships with Python 3.6 on Linux. v0.2.3 would be slightly more stable, but less features of course.

> I will create a `make.sh` script to preform all steps reported in various threads to automate build on FreeBSD and Linux.

I did make a CI/CD configuration from when no-one was developing UPBGE, but I had to work on a fork to attach the bots: https://github.com/bge-community/upbge/blob/master/appveyor.yml#L40-L67 maybe this can help.
